### PR TITLE
Quickjoin testing

### PR DESCRIPTION
**Pull request by kaadmy**
Adds a "Quick Join" feature to allow you to just click a button to drop you in the best server.

### CODE REVIEW

File: src/cgame/default/viewcontrollers/MultiplayerViewController.c [hunk 1]
```diff
@@ -31,6 +31,84 @@
...
+ * a bit of lovely random thrown in. Any server that matches the criteria will
+ * be weighted by how much "better" they are by how much lower their ping is and
+ * how many more players there are.
+ */
+static void quickjoinAction(Control *control, const SDL_Event *event, ident sender, ident data) {
+
+	const int16_t max_ping = Clamp(cg_quickjoin_max_ping->integer, 0, 999);
+	const int16_t min_clients = Clamp(cg_quickjoin_min_clients->integer, 0, MAX_CLIENTS);
+
+	GList *server = cgi.Servers();
```
Comment (jdolan) [inline src/cgame/default/viewcontrollers/MultiplayerViewController.c]: This is great, just name this variable "list" and name the `cl_server_info_t * server`.

File: src/cgame/default/viewcontrollers/MultiplayerViewController.c [hunk 2]
```diff
@@ -150,4 +229,3 @@ Class *_MultiplayerViewController(void) {
 }
 
 #undef _Class
-
```
Comment (jdolan) [inline src/cgame/default/viewcontrollers/MultiplayerViewController.c]: Make sure the file ends in newline.

Comment (kaadmy) [inline src/cgame/default/viewcontrollers/MultiplayerViewController.c]: It shows a newline for me in Emacs and Atom.

Comment (kaadmy) [inline src/cgame/default/viewcontrollers/MultiplayerViewController.c]: Just noticed that was a space that got removed, not the newline.

**Pull request merged**